visidata/deprecated.py

```diff
@@ -1,16 +1,20 @@
-from visidata import VisiData
+from visidata import Column, PyobjSheet, VisiData, vd
```

Comment:
For consistency with how it is in this file currently, and Reasons, let's keep this importing only `VisiData` and `vd`; reference the rest like `visidata.Column`.

Reply:
Sure thing, I support Reasons 😁 !